REQUIRE

```diff
@@ -1,8 +1,9 @@
 julia 0.4 0.5-
-Requires
 GZip 0.2.18
 StatsBase 0.7.4
 LightXML 0.2.1
 ParserCombinator 1.7.3
 JLD
 Clustering 0.4
+JuMP
```

Comment:
I am not in favor of this. It violates interdependence guideline 3 (non-core, non-leaf). The LG methods that require JuMP are not integral to the overall package (though their utility / benefit is not at issue).